cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,70 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Settings page', () => {
+  let user;
+  let newUser = {
+    password: 'ZAQ!2wsx',
+  };
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.login();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.visit();
+    settingsPage.typeUsername(user.username);
+    settingsPage.clickOnUpdateSettingsButton();
+    settingsPage.assertHeaderContainUsername(user.username);
+    settingsPage.visit();
+    settingsPage.usernameField.should('have.value', user.username);
```

Comment:
```suggestion
    settingsPage.usernameField.should('have.value', user.username);
```
wrap this line inside the method assertUsernameFieldHasValue()
Do this with all the same lines of code